src/snowflake/snowpark/mock/_functions.py

```diff
@@ -1700,42 +1700,53 @@ def mock_initcap(values: ColumnEmulator, delimiters: ColumnEmulator):
 
 @patch("convert_timezone")
 def mock_convert_timezone(
-    target_timezone: ColumnEmulator,
-    source_time: ColumnEmulator,
-    source_timezone: Optional[ColumnEmulator] = None,
+    *args: ColumnEmulator,
```

Comment:
Question: what's the motivation behind this change?

Reply:
Excerpt from functions.py:
```
    if source_timezone is None:
        return call_builtin("convert_timezone", target_tz, source_time_to_convert)
    return call_builtin(
        "convert_timezone", source_tz, target_tz, source_time_to_convert
    )
```
When convert_timezone is called with 2 parameters they are in a certain order, but when called with 3 the additional parameter is added to the front instead. In practice it seems that the mocked version always gets three parameters, but the ordering is different so I don't think it makes as much sense to name those parameters.